### PR TITLE
Indicate if the cache is still refreshing

### PR DESCRIPTION
**Pull request by GLundh**

### CODE REVIEW

File: src/main/resources/com/axis/system/jenkins/plugins/downstream/yabv/BuildFlowAction/buildFlowJsCss.groovy [hunk 1]
```diff
@@ -1,16 +1,20 @@
 package com.axis.system.jenkins.plugins.downstream.yabv.BuildFlowAction
 
+import com.axis.system.jenkins.plugins.downstream.cache.BuildCache
```
Comment (jonsten) [inline src/main/resources/com/axis/system/jenkins/plugins/downstream/yabv/BuildFlowAction/buildFlowJsCss.groovy]: Nit; seems unnecessary

File: src/main/resources/com/axis/system/jenkins/plugins/downstream/yabv/BuildFlowAction/buildFlowJsCss.groovy [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+      include(my, 'buildFlow.groovy')
+    }
+  }
   script("buildFlowRefreshInterval='${System.getProperty('yabv.buildFlowRefreshInterval', '10000')}'")
   script(src: "${rootURL}/plugin/yet-another-build-visualizer/scripts/render.js",
       type: 'text/javascript')
-  noscript() {
-    include(my, 'buildFlow.groovy')
-  }
+
```
Comment (jonsten) [inline src/main/resources/com/axis/system/jenkins/plugins/downstream/yabv/BuildFlowAction/buildFlowJsCss.groovy]: Nit; Remote?

File: src/main/resources/com/axis/system/jenkins/plugins/downstream/yabv/BuildFlowAction/buildFlowJsCss.groovy (hunk 1 above)
Comment (GLundh) [reply src/main/resources/com/axis/system/jenkins/plugins/downstream/yabv/BuildFlowAction/buildFlowJsCss.groovy]: Why?

Comment (jonsten) [reply src/main/resources/com/axis/system/jenkins/plugins/downstream/yabv/BuildFlowAction/buildFlowJsCss.groovy]: It is never used in this file.

Comment (GLundh) [reply src/main/resources/com/axis/system/jenkins/plugins/downstream/yabv/BuildFlowAction/buildFlowJsCss.groovy]: God. I don't now how it is possible, but 3 times I opened the wrong file and double checked that it was actually used. Sorry about that.

**Pull request merged**